build-ipa.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -ex
+
+TMPDIR=$(mktemp -d -p /var/tmp)
+TMPDIR_RAMDISK=$(mktemp -d -p /var/tmp)
+
+chmod 755 $TMPDIR
+cd $TMPDIR
+tar -xf /usr/share/rhosp-director-images/ironic-python-agent-latest.tar
+cd $TMPDIR_RAMDISK
+/usr/lib/dracut/skipcpio $TMPDIR/ironic-python-agent.initramfs | zcat | cpio -ivd
+rpm2cpio /tmp/packages/openstack-ironic-python-agent*.rpm | cpio -ivdum
+rpm2cpio /tmp/packages/python3-ironic-python-agent*.rpm | cpio -ivdum
```

Comment:
While we're here, you probably need ironic-lib as well

Reply:
I guess it makes sense, it would just mean tagging one more package in the end